core/templates/pages/community-dashboard-page/community-dashboard-page.mainpage.html

```diff
@@ -20,6 +20,6 @@
 
     </oppia-root>
     @load('pages/footer_js_libs.html')
-    <script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=TeX-AMS_SVG"></script>
+    <script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=default"></script>
```

Comment:
Why are you changing this? I think there should be no change if we are not updating MathJax?

Ditto below.

Reply:
This just changes the config file used while typesetting. I'm changing this because the previous configuration caused an issue while typesetting a very large equation.
Moreover config=default does not interfere with the custom config file we created while the previous setting sometimes does.